httpsender/Alert on HTTP Response Code Errors.js

```diff
@@ -0,0 +1,50 @@
+// A script which will raise alerts based on HTTP Response codes
+// By default it will raise 'Info' level alerts for Client Errors (4xx) and 'Low' Level alerts for Server Errors (5xx)
+// But it can be easily changed.
+
+// The following handles differences in printing between Java 7's Rhino JS engine
+// and Java 8's Nashorn JS engine
+if (typeof println == 'undefined') this.println = print;
+
+pluginid = 100000	// https://github.com/zaproxy/zaproxy/blob/develop/src/doc/scanners.md
+
+function sendingRequest(msg, initiator, helper) {
+	// Nothing to do
+}
+
+function responseReceived(msg, initiator, helper) {
+	var extensionAlert = org.parosproxy.paros.control.Control.getSingleton().getExtensionLoader().getExtension(
+		org.zaproxy.zap.extension.alert.ExtensionAlert.class)
```

Comment:
I'm getting an error when running with Rhino, `EvaluatorException: missing name after . operator (<Unknown source>#17)`.
It works fine with both engines when used the extension name.